### ISSUE TITLE
Population script improvements

### ISSUE THREAD

**Issue by ckoerber** [callat-qcd/espressodb#28]
Here are a few ideas how to improve the population script view:

- [x] `instance = cls.get_or_create(...)` should be `instance, created = cls.objects.get_or_create(...)`
- [x] Add help text to the current selection (and tree table)
- [ ] Backwards / undo option
- [x] Autofill foreign key options (as the name of the variable is already predefined before
- [x] Remove three dots in kwargs (do not want to remove it each time when filling out)

**Comment by ckoerber** [comment]
Next update: Adjust script for many-to-many columns.

It is currently rendered as
```
ensemble_ensemble, created = ensemble_Ensemble.objects.get_or_create(
    label=, # (Optional) Char(20): label to identify ensemble for easy searches
    configurations=, # 
    tag=, # (Optional) User defined tag for easy searches
)
```

Remove the config key here and add a line according to
```
ensemble_ensemble, created = ensemble_Ensemble.objects.get_or_create(
    label=, # (Optional) Char(20): label to identify ensemble for easy searches
    tag=, # (Optional) User defined tag for easy searches
)
ensemble_ensemble.add(configuration1, configuration2,...)
```

**Comment by ckoerber** [comment]
Also, when a FK column points to just one table, this table is automatically selected. However, it might be the case that the user does not want to find dependencies for this entry, which is not the default behavior. An easy fix would be a global check box what to do with dependencies. Another option, which removes default selection, would be to ask for a given entry if dependencies should be found.

Addressing this is less urgent though.

**Comment by ckoerber** [comment]
For now we will not include the FK and M2M updates in v1.0.0.

**Comment by ckoerber** [comment]
Closed with #37
